datarepo-clienttests/src/main/java/scripts/testscripts/RetrieveDataset.java

```diff
@@ -21,7 +21,7 @@ public void userJourney(TestUserSpecification testUser) throws Exception {
     ApiClient apiClient = DataRepoUtils.getClientForTestUser(testUser, server);
     RepositoryApi repositoryApi = new RepositoryApi(apiClient);
     DatasetModel datasetModel = repositoryApi.retrieveDataset(datasetSummaryModel.getId());
-    logger.debug(
+    logger.info(
```

Comment:
Not sure what the distinction of INFO vs DEBUG is for test code. My expectation is that a test will be as silent as possible unless an error occurs. One way to do this is to capture logs and only output them if the test fails. It's less useful IMO to use different log levels like you would in an application code.

Reply:
Just that we never run in debug mode so it seems silly to not just put it in info mode (also, less flying blind)